requirements.txt

```diff
@@ -1,9 +1,13 @@
 PyYAML
 ansi2html
+-e git+https://github.com/hroncok/elsa.git@frozen_flask_warnings#egg=elsa
```

Comment:
I just got around to the review there. Are you planning to release Elsa 0.1.3 for this?